src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -319,33 +336,14 @@ public void OnBindingStateChanged_NewConfiguration_EventRaised()
 
                 // Assert
                 // Different config so event should be raised
-                solutionBindingChangedEventCount.Should().Be(1);
-            }
-        }
-
-        [TestMethod]
-        public void OnBindingStateChanged_SameConfiguration_EventNotRaised()
```

Comment:
Since we're now checking all of the event counts in each test, this test is now redundant (there's another test that does the same thing below).